benches/compressed-snark.rs

```diff
@@ -43,7 +43,7 @@ cfg_if::cfg_if! {
     criterion_group! {
       name = compressed_snark;
       config = Criterion::default().warm_up_time(Duration::from_millis(3000));
-      targets = bench_compressed_snark, bench_compressed_snark_with_computational_commitments
+    targets = bench_compressed_snark, bench_compressed_snark_with_computational_commitments,
```

Comment:
Nit: this space seems to be removed, right? I'm surprised `cargo fmt` doesn't seem to catch this.

Reply:
oh, weird. will fix.